### DISCUSSION TITLE
Add New Exit Type

### DISCUSSION THREAD

**Discussion by fexhvhbdsf864** [Wohlstand/TheXTech#530] [Ideas]
My Idea is:
- [x] Flag Pole (SMB) (Already Implement)
- [ ] Flag Pole (Secret, NSMB)
- [ ] Mask Gate (SMB2)
- [ ] Goal (Invisible, SML)
- [ ] Bonus Bell (SML2)
- [ ] Goal Post (Sonic)
- [ ] Animal Capsule (Sonic)
- [ ] Goal Ring (NSMBU)
- [ ] Goal Ring (Sonic)
etc.

**Comment by LoveBodhi** [comment]
- [ ] SMW2 (YI) Goal Ring
- [ ] WL4 Frog Switch?
- [ ] Kirby Jumping-based Goal

**Comment by vitzie629** [comment]
When it comes to the flagpole, there would be a normal, snow, autumn, & more variants of the pole BGOs. Also, it would have the ability to either put the flag on the left or the right of the pole.

**Comment by Wohlstand** [comment]
... I should move this into discussions too. New exit types will can be added in only condition when new exit mechanisms will be implemented somewhere as a part of extended content.

**Comment by LoveBodhi** [comment]
Actually, part of #TheXTech Asset Pack# topic...?
